setup.py

```diff
@@ -19,7 +19,7 @@
 
 requirements = [
     'graphene-sqlalchemy>=2.1.0,<3',
-    'SQLAlchemy<2',
+    'SQLAlchemy==1.3.23',
```

Comment:
I'm not a big fan of hard requirements like this. 

Are you using a specific features of SQLAlchemy `1.3.23` ? Can we make that `SQLAlchemy>=1.3,<2` instead ?

Reply:
Yes you're right, but unfortunately there is some stuff in the library (that is unrelated to this PR) that now fails due to the recent release of sqlalchemy 1.4, therefore I limited it to 1.3.23 for the time being. I have to see if maybe #54 fixes those issues already.

Reply:
How about this then ?

```suggestion
    'SQLAlchemy>=1.3,<1.4',
```

Reply:
I resolved the failing tests (some of it was already fixed on main). It now runs through with the latest SQLAlchemy > 1.3.23.

Reply:
Thanks.